### DISCUSSION TITLE
Is PISM able to do global simulation?

### DISCUSSION THREAD

**Discussion by jacksd233** [pism/pism#537] [General]
Hi,
I am wondering if PISM can do a global simulation? 
it seems that most works using PISM are focusing on specific ice sheets under rectangular regions.

**Comment by crodehacke** [comment]
Hej Weiwen Ji,

PISM generally uses rectangular (orthogonal) grids, meaning a global simulation would have to utilize such a grid. This is hard to achieve, considering the global sphere.

In addition, some processes, such as those related to Glacial Isostatic Adjustment (GIA), require for the viscous part, an extended grid, which may inhibit a simple global spherical grid. Also, some parameterizations related to the calving processes, require even squared grids. Therefore, I guess it is nothing other than trivial to run PISM globally while expecting that all processes could be used. At least a so-called _lon-lat_ grid is not supported to my knowledge. Best, Christian

**Comment by jacksd233** [comment]
@crodehacke Got it, thanks!

**Comment by talbrecht** [comment]
Hej  Weiwen Ji,
in the Earth System Model context people often use two or three instances of PISM for the different regions (e.g. Antarctica and Northern Hemisphere) using stereographic projections. But if you have a Snowball Earth simulation in mind, this may be a different challenge...
Torsten
